docs/shared/core-features/automate-updating-dependencies.md

```diff
@@ -41,24 +45,22 @@ At this stage, after inspecting the `package.json`, you may wish to manually run
 
 ### Step 2: Running migrations
 
-The next step in the process involves using the `migrate` CLI in order to apply the migrations that were generated in `migrations.json` in the previous step.
-
-Each Nx plugin is able to provide a set of migrations which are relevant to particular versions of the package, and so `migrations.json` will only contain migrations which are appropriate for the update you are currently applying.
-
-The common case is that you will simply apply all migrations from the generated JSON file, exactly as they were generated in the previous step, by running:
+You can now run the actual code migrations that were generated in the `migrations.json` in the previous step.
 
 ```shell
 nx migrate --run-migrations
 ```
 
 This will update your source code in your workspace in accordance with the implementation of the various migrations which ran and all the changes will be unstaged ready for you to review and commit yourself.
 
+Note that each Nx plugin is able to provide a set of migrations which are relevant to particular versions of the package. Hence `migrations.json` will only contain migrations which are appropriate for the update you are currently applying.
+
 ### Step 3: Cleaning up
 
 After you run all the migrations, you can remove `migrations.json` and commit any outstanding changes.
 
 Note: You may want to keep the `migrations.json` until every branch that was created before the migration has been merged. Leaving the `migrations.json` in place allows devs to run `nx migrate --run-migrations` to apply the same migration process to their newly merged code as well.
 
-## Problems?
+## Need to opt-out of some migrations?
 
-If you can't run `nx migrate --run-migrations` all in one step, try the tips in [Advanced Update Process](/recipes/other/advanced-update)
+Sometimes you need to temporarily opt-out from some migrations because your workspace is not ready yet. As mentioned, you can either manually adjust the `migrations.json` or run the update with the `--interactive` flag. More details in our [Advanced Update Process](/recipes/other/advanced-update) guide.
```

Comment:
```suggestion
Sometimes you need to temporarily opt-out from some migrations because your workspace is not ready yet. You can manually adjust the `migrations.json` or run the update with the `--interactive` flag to choose which migrations you accept. Find more details in our [Advanced Update Process](/recipes/other/advanced-update) guide.
```